x-pack/legacy/plugins/siem/server/lib/detection_engine/scripts/rules/queries/query_with_errors.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Query with a rule id",
+  "description": "Query with a rule_id that acts like an external id",
```

Comment:
I would change the description to match the rule test case of: `Query with a rule_id that causes an error`